macros/GroupData.json

```diff
@@ -1721,6 +1721,55 @@
             "properties": [],
             "events":     []
         },
+        "WebXR Device API": {
```

Comment:
I am opposed to changing this. The WebXR spec has been broken into multiple parts and it's important to be clear which this is.

Reply:
In that case, the relevant pages should be updated.

Reply:
Yes, certainly so. Review and updates to the pages that have been contributed so far will be the next thing I do on WebXR content, coming very soon  (hopefully starting next sprint).

Given this agreement (I suppose that's what it is anyway), this PR is still awaiting final review and approval.

Reply:
If I used 'WebXR API' on the pages I created, then I shouldn't have. The spec itself has been the 'WebXR Device API' [since the very beginning](https://www.specref.org/?q=WebXR)

Reply:
I believe this comment is resolved then; "WebXR Device API" it is. Can this be merged?